app/graphql/types/query_type.rb

```diff
@@ -7,5 +7,6 @@ class QueryType < Types::BaseObject
     include GraphQL::Types::Relay::HasNodesField
 
     field :freaks, resolver: Resolvers::FreaksResolver
+    field :norms, resolver: Resolvers::NormsResolver
```

Comment:
Where's the spec for this?

Reply:
![image](https://user-images.githubusercontent.com/2707601/130189529-70740195-dd32-425f-805f-b533278468ff.png)